snapcraft.yaml

```diff
@@ -1,23 +1,23 @@
 name: core18
-# version: 18
-adopt-info: boostrap
+adopt-info: bootstrap
 summary: Runtime environment based on Ubuntu 18.04
 description: |
   The base snap based on the Ubuntu 18.04 release.
 confinement: strict
 type: base
+build-base: core18
```

Comment:
this should be redundant for snaps of type:base as the `name` is assumed as the base by default.

And i thought setting build-base on type:base is not supported and makes the build fail. Or has that been fixed in snapcraft now?

Reply:
it is successful in travis without complaints, interesting.

Reply:
hopefully this will not fail in launchpad.

Reply:
>this should be redundant for snaps of type:base as the name is assumed as the build-base by default.

I was not aware of this fact and at least when I opened this PR it was not the case, maybe it is now the case today...

> And i thought setting build-base on type:base is not supported and makes the build fail. Or has that been fixed in snapcraft now?

this is contrary to what I have heard, I thought that specifying build-base is the proper way to specify this now...